### PR TITLE
contribute feature docs

### PR DESCRIPTION
**Pull request by parteekcoder**
Hi @pradeeban ,

I started writing docs this week. I will be glad if you provide reviews of this PR so I can improve it further.

### CODE REVIEW

File: docs/source/concoreexample.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Concore examples
```
Comment (pradeeban) [inline docs/source/concoreexample.rst]: This should be named as something else, and not "Concore examples."

Perhaps, "The Contribute Action?"

File: docs/source/index.rst [hunk 2]
```diff
@@ -24,4 +24,5 @@ Contents
    usage
    raspberrypi
    concoreeditor
+   concoreexample
```
Comment (pradeeban) [inline docs/source/index.rst]: rename as contributeaction

File: docs/source/concoreexample.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+.. image:: images/ce-contribute-loading.png
+  :width: 700
+
+- After it gets completed, you can check at the link given in the message
+
+.. image:: images/ce-contribute-success.png
+  :width: 700
+
+
+``Note: Successful submission doesnot mean that your example is added to our repository, It will take time for us to review and accept the example.So, you can mention your email in the description field mentioned above so that if your example got accepted then we will let you know.``
```
Comment (pradeeban) [inline docs/source/concoreexample.rst]: Rename "example" here with "study"

Don't use "example" in this feature. We/users are not contributing "examples." They are contributing studies.

**Comment by pradeeban** [comment]
I have left a few comments. Please address them.

File: docs/source/concoreexample.rst (hunk 3 above)
Comment (parteekcoder) [reply docs/source/concoreexample.rst]: Yeah I will update that, thanks for pointing it out

**Comment by parteekcoder** [comment]
Thanks @pradeeban for pointing these mistakes. I will correct them , and also add the docs for how run it using contribute script

**Pull request merged**